package.json

```diff
@@ -514,17 +514,17 @@
                   "items": {
                     "type": "string"
                   },
-                  "description": "Array of symbol server URLs (example: http\u200b://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
+                  "description": "Array of symbol server URLs (example: http​://MyExampleSymbolServer) or directories (example: /build/symbols) to search for .pdb files. These directories will be searched in addition to the default locations -- next to the module and the path where the pdb was originally dropped to.",
```

Comment:
This happened automatically. Should I revert?

Reply:
If you could, that would help me spin up a new pre-release. There is a launch configuration `Update Package Dependencies` which I run when I update OmniSharp, but also when I want to add back the zero-length spaces.

Reply:
If you run `npm run gulp generateOptionsSchema` it should put it back. See src\tools if you need more info.

Reply:
Going to put that in my bag of tricks. =)

Reply:
Done